deep_tempering/training_utils.py

```diff
@@ -14,6 +14,11 @@
 LOGS_PATH = os.path.join(os.getcwd(), '.deep_tempering_logs')
 
 
+_IS_TRAINING_PLACEHOLDER = tf.compat.v1.placeholder_with_default(True, shape=())
```

Comment:
Let's change it to this, otherwise we might have bug with non-default sessesions:
```python
with tf.compat.v1.keras.backend.get_session().graph.as_default():
  _IS_TRAINING_PLACEHOLDER = tf.compat.v1.placeholder_with_default(True, shape=())
```